src/platforms/common/session-replay/custom-instrumentation/privacy-configuration.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Privacy Configuration
+sidebar_order: 4200
+description: "Configuring Session Replay to maintain user and data privacy."
+---
+
+There are several ways to deal with PII (personally identifiable information). By default, the integration will mask all text content with `*` and block all media elements (`img, svg, video, object, picture, embed, map, audio`). This can be disabled by setting `maskAllText` to `false`. It is also possible to add the following CSS classes to specific DOM elements to prevent recording its contents: `sentry-block`, `sentry-ignore`, and `sentry-mask`. The following sections will show examples of how content is handled by the differing methods.
+
+### Masking
+Masking replaces the text content with something else. The default masking behavior is to replace each character with a `*`. In this example the relevant html code is: `<table class="sentry-mask">...</table>`.
+![Masking example](session-replay-masking.png)
+
+
+### Blocking
+Blocking replaces the element with a placeholder that has the same dimensions. The recording will show an empty space where the content was. In this example the relevant html code is: `<table data-sentry-block>...</table>`.
+![Blocking example](session-replay-blocking.png)
+
+
+### Ignoring
+Ignoring only applies to form inputs. Events will be ignored on the input element so that the replay does not show what occurs inside of the input. In the below example, notice how the results in the table below the input changes, but no text is visible in the input.
+
+![Ignoring Example](session-replay-ignore-input.mp4)
```

Comment:
This isn't playing, it was originally a `.mov` file, i renamed it because some googling suggested that... it didn't work

Next approach: convert it to gif